content/en/tracing/trace_collection/custom_instrumentation/java/otel.md

```diff
@@ -181,6 +181,55 @@ public class Example {
 }
 ```
 
+## Adding span events
+
+_Minimum SDK version: 1.40.0._
+
+You can add span events using the `addEvent` API. This method requires a `name` parameter and optionally accepts `attributes` and `timestamp` parameters. The method creates a new span event with the specified properties and associates it with the corresponding span.
+
+- **Name** [_required_]: A string representing the event's name.
+- **Attributes** [_optional_]: Zero or more key-value pairs with the following properties:
+  - The key must be a non-empty string.
+  - The value can be either:
+    - A primitive type: string, Boolean, or number.
+    - A homogeneous array of primitive type values (for example, an array of strings).
+  - Nested arrays and arrays containing elements of different data types are not allowed.
+- **Timestamp** [_optional_]: A UNIX timestamp representing the event's occurrence time, expects an `Instant` object.
```

Comment:
```suggestion
- **Timestamp** [_optional_]: A UNIX timestamp representing the event's occurrence time. Expects an `Instant` object.
```

Reply:
Than you! Updated in https://github.com/DataDog/documentation/pull/25157/commits/9e2c256728da10da5a7a200c66fbcd8463d9c43a.